cfme/tests/cloud/test_instance_power_control.py

```diff
@@ -60,6 +58,39 @@ def _wait_for_state_refresh():
         return False
 
 
+def wait_for_termination(provider, instance):
+    """ Waits for VM/instance termination and refreshes power states and relationships
+    """
+    state_change_time = instance.get_detail(('Power Management', 'State Changed On'))
+    provider.refresh_provider_relationships()
+    logger.info("Refreshing provider relationships and power states")
+    refresh_timer = RefreshTimer(time_for_refresh=300)
+    wait_for(provider.is_refreshed,
+             [refresh_timer],
+             message="is_refreshed",
+             num_sec=1000,
+             delay=60,
+             handle_exception=True)
+    wait_for_state_change_time_refresh(instance, state_change_time, timeout=720)
+    if instance.get_detail(('Power Management', 'Power State')) not in \
+            {'terminated', 'archived', 'unknown'}:
```

Comment:
`{instance.STATE_TERMINATED,  instance.STATE_ARCHIVED, instance.STATE_UNKNOWN}` instead of strings directly because it can change in the future and it can potentially differ between instance types.